x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/100_tsdb.yml

```diff
@@ -85,6 +85,32 @@ aggretate multi_terms:
   - match: { aggregations.m_terms.buckets.2.key_as_string: "{k8s.pod.uid=947e4ced-1786-4e53-9e0c-5c447e959507, metricset=pod}|10.10.55.2" }
   - match: { aggregations.m_terms.buckets.2.doc_count: 1 }
 
+---
+"multi_terms aggregation with time_series aggregation":
+  - skip:
+      version: " - 8.12.99"
+      reason: "multi_terms for time series aggregation fixed in 8.13.0"
+
+  - do:
+      search:
+        index: test
+        body:
+          aggs:
+            ts:
+              time_series: {}
+            m_terms:
+              multi_terms:
+                collect_mode: breadth_first
```

Comment:
In this case we force `depth_first` no matter the fact that the user requested `breadth_first`...wouldn't it be better to fail and tell the user we don't support combining multi terms with `depth_first` and time series agg?

Reply:
We could do this, but then we would need to change this in terms agg too. I like to keep this consistent.
We can however change this in a followup pr, if we feel strongly that both multi_terms and terms agg should fail in this case.